examples/build_scripts/build_gaea.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Example bash script to build Pace to run bare-metal on Gaea's c4 cluster
+
+set -e -x
+
+# module load necessary system software
+module rm PrgEnv-intel
+module load PrgEnv-gnu
+module rm gcc
+module load gcc/10.3.0
+module load boost/1.72.0
+module load python/3.9
+
+# set environment variables
+export MPICC=$CC
```

Comment:
What is this needed for? I don't think any of the commands below trigger compilaton?